website/site/content/platforms/cgw.md

```diff
@@ -0,0 +1,161 @@
+---
+title: "CGW"
+description: "See what you can do with the CGW: all implemented sensors and connectivity."
+weight: 40
+draft: false
+toc: true
+menu:
+  main:
+    parent: Platforms
+    identifier: CGW
+    weight: 0
+---
+
+
+## Overview: The CGW
+
+The Common GateWay is a Bosch internal development platform, equipped with BMA280, BME280, BLE, CAT-M1/NB-IoT and GPS.
```

Comment:
The Common GateWay (CGW) is a Bosch.. etc.

Reply:
```suggestion
The Common GateWay (CGW) is a Bosch internal development platform, equipped with BMA280, BME280, BLE, CAT-M1/NB-IoT and GPS.
```